iroha-cli/main.cpp

```diff
@@ -17,13 +17,14 @@
 
 #include <gflags/gflags.h>
 #include <rapidjson/istreamwrapper.h>
-#include <rapidjson/rapidjson.h>
```

Comment:
I suggest not removing this include, even if it works without it, because we directly create and use `rapidjson::Document` in this file.